extensions/vertx/deployment/src/test/java/io/quarkus/vertx/deployment/MessageConsumerFailureTest.java

```diff
@@ -35,18 +38,24 @@
 
     @Test
     public void testFailure() throws InterruptedException {
-        verifyFailure("foo", "java.lang.IllegalStateException: Foo is dead");
-        verifyFailure("foo-message", "java.lang.NullPointerException");
-        verifyFailure("foo-completion-stage", "java.lang.NullPointerException: Something is null");
+        verifyFailure("foo", "java.lang.IllegalStateException: Foo is dead", false);
+        verifyFailure("foo-message", "java.lang.NullPointerException", false);
+        verifyFailure("foo-completion-stage", "java.lang.NullPointerException: Something is null", false);
+        verifyFailure("foo-completion-stage-failure", "boom", true);
+        verifyFailure("foo-uni", "java.lang.NullPointerException: Something is null", false);
+        verifyFailure("foo-uni-failure", "boom", true);
     }
 
-    void verifyFailure(String address, String expectedMessage) throws InterruptedException {
+    void verifyFailure(String address, String expectedMessage, boolean explicit) throws InterruptedException {
         BlockingQueue<Object> synchronizer = new LinkedBlockingQueue<>();
+        System.out.println("sending to " + address);
```

Comment:
Remove any `System.outs ` from the code (including tests)

Reply:
Thanks! I don't even remember adding these traces. They have been removed.